tests/e2e_args.py

```diff
@@ -90,6 +90,12 @@ def cli_args(add=lambda x: None, parser=None, accept_unknown=False):
     parser.add_argument(
         "--consensus", help="Consensus", default="raft", choices=("raft", "pbft"),
     )
+    parser.add_argument(
+        "--worker_threads",
+        help="number of worker threads inside the enclave",
+        type=int,
+        default=1,
```

Comment:
Default for worker threads elsewhere is 0, is there a reason for it to be 1 here?